### ISSUE TITLE
Allow adjusting padding spaces in the list markup

### ISSUE THREAD

**Issue by Lin-Buo-Ren** [0x6b/copy-selection-as-markdown#51]
**Actual Result**

```
/* --- copy-selection-as-markdown debug information --- */ copy.js:54:14
/* --- INPUT ------------------------------------------ */ copy.js:55:14
<li>Manage the life cycle of docker containers.</li>
<li>Supports check mode. Run with <code class="docutils literal notranslate"><span class="pre">--check</span></code> and <code class="docutils literal notranslate"><span class="pre">--diff</span></code> to view config difference and list of actions to be taken.</li> copy.js:56:14
/* --- OUTPUT------------------------------------------ */ copy.js:57:14
-   Manage the life cycle of docker containers.
-   Supports check mode. Run with `--check` and `--diff` to view config difference and list of actions to be taken. copy.js:58:14
/* --- URL -------------------------------------------- */ copy.js:59:14
https://docs.ansible.com/ansible/latest/modules/docker_container_module.html#docker-container-module copy.js:60:14
/* ---------------------------------------------------- */
```

**Expected Result**

Allow customizing  the quantity of the padding spaces from three to one so the resulting output can be adjusted as the following:

```
- Manage the life cycle of docker containers.
- Supports check mode. Run with `--check` and `--diff` to view config difference and list of actions to be taken. copy.js:58:14
```

**Comment by 0x6b** [comment]
Thanks @Lin-Buo-Ren for the request. Such feature keeps popping up in my mind. Let me see how I can implement it.

**Comment by 0x6b** [comment]
Hey @Lin-Buo-Ren  I just released v0.17.0 which implements your request -- "Reduce padding spaces for list items." Grab the new from Add-on page at your Firefox setup. Open an issue if you find any problem. Thanks!
